.vscode/tasks.json

```diff
@@ -0,0 +1,24 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build",
+            "command": "dotnet",
+            "type": "shell",
+            "args": [
+                "build",
+                // Ask dotnet build to generate full paths for file names.
+                "/property:GenerateFullPaths=true",
```

Comment:
I might steal these for the auto-generated tasks that Ionide creates for projects - they're a nice QoL change!

Reply:
This was actually the template VS Code provided, but it is a good one :)